labcodes/labstorybook
# DSYS-197: Write dialogs docs

**Link to task**
https://labcodes.atlassian.net/browse/DSYS-197

**How to test**
With the labsystem running, go to the `Dialog` tab and check if everything is according to the documentation.
See https://www.figma.com/file/Wx7jj5klQn3YsRA2XcIV3E/%F0%9F%93%9A-Components?node-id=8904%3A2123 for a detailed reference.

**Description**
This PR translates `Dialog`'s documentation from Figma to Storybook. 
Some adjustments are still required before merging. I leave here points for consideration:
- Are the correct tags used for each section and subsection?
- Is there a CSS for a white box with red letters (see subsection "Sizes")?
- Is there a CSS for vertical lines?
- Are the pictures OK?
